src/DGtal/io/Display3D.h

```diff
@@ -424,7 +424,7 @@ namespace DGtal
      * @return the select callback function that match the given \a
      * name, or 0 if none is associated to this name.
      */
-    SelectCallbackFct getSelectCallback3D( DGtal::int32_t name, void*& data ) const;
+    SelectCallbackFct getSelectCallback3D( DGtal::int32_t aName, void*& data ) const;
```

Comment:
Need to update the comment above "name" => "aName".
That's why clang fail on travis.